.github/workflows/pull-request-test.yml

```diff
@@ -86,20 +86,25 @@ jobs:
         KUBECONFIG: ${{ github.workspace }}/renkubot-kube.config
         RENKUBOT_KUBECONFIG: ${{ secrets.RENKUBOT_KUBECONFIG }}
         CI_RENKU_VALUES: ${{ secrets.CI_RENKU_VALUES }}
+        RENKUBOT_RANCHER_APISECRET: ${{ secrets.RENKUBOT_RANCHER_APISECRET }}
       run: |
         echo "$RENKUBOT_KUBECONFIG" > renkubot-kube.config
-        helm init --client-only
+        helm repo add stable 	https://kubernetes-charts.storage.googleapis.com
         helm repo add renku https://swissdatasciencecenter.github.io/helm-charts/
         helm repo add gitlab https://charts.gitlab.io/
         helm repo add jupyterhub https://jupyterhub.github.io/helm-chart
         rm -rf charts/renku/charts
         helm dependency update charts/renku
         printf "%s" "$CI_RENKU_VALUES" | sed "s/<replace>/${RENKU_RELEASE}/" > values.yaml
-        kubectl create namespace $RENKU_NAMESPACE || true
+        curl -u "token-fxgsb:$RENKUBOT_RANCHER_APISECRET" \
+        -X POST \
+        -d "name=${RENKU_NAMESPACE}" \
+        -d "projectId=c-l6jt4:p-rz88m" \
```

Comment:
I was wondering if we want to hide user and project ID, but I actually think it's fine.

Reply:
yeah I thought about that too, I can add them to the secrets. Just FYI, here we have the cluster and project ID, not user ID.